package.json

```diff
@@ -20,83 +22,58 @@
   },
   "scripts": {
     "ng": "ng",
+    "prestart": "npm run prestart --workspace=packages --if-present",
     "start": "ng serve --hmr --proxy-config proxy.conf.json",
-    "start.watch": "run-p watch.libs \"_start.watch -- -- {@}\" --",
     "build": "ng build",
     "lint": "ng lint",
     "lint.fix": "ng lint --fix",
-    "e2e": "ng run demo:e2e:production",
+    "e2e": "npm run prestart && ng run demo:e2e:production",
     "e2e.local": "ng run demo:e2e:local",
     "build.all": "npm run build.libs && npm run build.demo",
-    "build.libs": "rimraf ./dist && npm run build.utils && npm run build.core && npm run build.common && npm run build.auth && npm run build.geo && npm run build.context && npm run build.integration",
-    "build.utils": "ng build utils --configuration production",
-    "build.core": "ng build core --configuration production && npm run build.locale",
-    "build.common": "ng build common --configuration production",
-    "build.auth": "ng build auth --configuration production",
-    "build.geo": "ng build geo --configuration production",
-    "build.context": "ng build context --configuration production",
-    "build.integration": "ng build integration --configuration production",
```

Comment:
I propose adopting the Workspace features and move the scripts into their package. This will clean the root package scripts

Reply:
good.